service/version_bundles.go

```diff
@@ -0,0 +1,78 @@
+package service
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func newVersionBundles() []versionbundle.Bundle {
+	return []versionbundle.Bundle{
+		{
+			Changelogs: []versionbundle.Changelog{
+				{
+					Component:   "kubernetes",
+					Description: "enable encryption at rest",
+					Kind:        versionbundle.KindAdded,
+				},
+				{
+					Component:   "kubernetes",
+					Description: "update to version 1.9.0",
+					Kind:        versionbundle.KindChanged,
+				},
+				{
+					Component:   "kubernetes",
+					Description: "use vanilla (previously coreos) hyperkube image",
+					Kind:        versionbundle.KindChanged,
+				},
+				{
+					Component:   "calico",
+					Description: "update to version 3.0.1",
+					Kind:        versionbundle.KindChanged,
+				},
+				{
+					Component:   "calico",
+					Description: "remove calico-ipip-pinger",
+					Kind:        versionbundle.KindRemoved,
+				},
+				{
+					Component:   "calico",
+					Description: "remove calico-node-controller",
+					Kind:        versionbundle.KindRemoved,
+				},
+			},
```

Comment:
I followed k8scloudconfig changelog: https://github.com/giantswarm/k8scloudconfig/blob/52cd0230d6fb7346b7b33b75e74e8e12990def21/CHANGELOG.md

And http://keepachangelog.com/en/1.0.0/ format